Jenkinsfile_CNP

```diff
@@ -193,6 +193,11 @@ withPipeline(type, product, component) {
       setCommonEnvVariables()
       setFeatureFlags(env.ENVIRONMENT)
 
+      def githubApi = new GithubAPI(this)
+      if (!githubApi.getLabelsbyPattern(env.BRANCH_NAME, "keep-helm")) {
+        enableCleanupOfHelmReleaseOnSuccess()
```

Comment:
Not sure you meant to make this change? 
I think you need to rebase?

Reply:
It was already on the master branch. I removed it because of failed jenkins check (that's what not what I need to fix) and re-add it.